Sources/SIL/Bitstream.swift

```diff
@@ -45,15 +46,6 @@ struct Bits: Equatable, Hashable, ExpressibleByIntegerLiteral, CustomStringConve
         return Int(asUInt32())
     }
 
-    func asString() -> String? {
```

Comment:
"asXXX" is not a common pattern in Swift APIs. Usually we'd define a computed property with a noun as the name.

Reply:
You can also use the [`CustomStringConvertible`](https://developer.apple.com/documentation/swift/customstringconvertible) for this particular use case with a field called `description`.